test/integration/aws/default/build/rds.tf

```diff
@@ -0,0 +1,25 @@
+# Contains resources and outputs related to testing the aws_rds_* resources.
+
+#======================================================#
+#                    RDS Instances
+#======================================================#
+
+
+resource "aws_db_instance" "default" {
```

Comment:
This is just a preference, but I like to name test fixtures with a more useful name - perhaps "alpha".  I just don't like the word "default", when there isn't anything default about it.